From ebbc468b211201838db7675a2ef9d2611c5c6ee4 Mon Sep 17 00:00:00 2001
From: Antonio Larrosa <antonio.larrosa@gmail.com>
Date: Mon, 18 Mar 2019 17:13:28 +0100
Subject: [PATCH 1/3] Hide updates and checkbox when system is offline.

Hide actions that can't be taken if the system doesn't have a network
connection.
---
 src/plasma/contents/ui/Full.qml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/plasma/contents/ui/Full.qml b/src/plasma/contents/ui/Full.qml
index eca7ec0..7cf37eb 100644
--- a/src/plasma/contents/ui/Full.qml
+++ b/src/plasma/contents/ui/Full.qml
@@ -127,7 +127,7 @@ Item {
             id: updatesScrollArea
             Layout.fillWidth: true
             Layout.fillHeight: true
-            visible: PkUpdates.count && !PkUpdates.isActive
+            visible: PkUpdates.count && PkUpdates.isNetworkOnline && !PkUpdates.isActive
 
             ListView {
                 id: updatesView
@@ -160,7 +160,7 @@ Item {
         }
 
         RowLayout {
-            visible: PkUpdates.count && !PkUpdates.isActive
+            visible: PkUpdates.count && PkUpdates.isNetworkOnline && !PkUpdates.isActive
             PlasmaComponents.CheckBox {
                 id: chkSelectAll
                 anchors {
-- 
2.21.0

